user/src/styles/pages/index.module.scss

```diff
@@ -0,0 +1,32 @@
+
+
+.structure {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 200px;
+  justify-content: center;
+  left: 302px;
+  
+  
+}
+.content{
+  
+  
+  color:rgba(40, 141, 122, 1)
+  
+}
+.title{
+  color:rgba(79, 79, 79, 1);
+  font-weight: 2000;
+}
+.button{
+  color:rgba(31, 100, 133, 1);
+  border-radius: 8px;
```

Comment:
Instead of hardcoding colors use the defined global variables, like for example:

```css
.my-class {
  color: var(--color-primary); /* Use the global primary color in the `color` property of the class */
}
```

To see the complete list of available CSS and SASS global variables open `src/styles/theme.ts`.